src/fontMetrics.js

```diff
@@ -252,27 +251,30 @@ const getCharacterMetrics = function(
     } else if (cjkRegex.test(character[0])) {
         ch = 'M'.charCodeAt(0);
     }
-    const metrics = metricMap[font][ch];
+    const metrics = metricMap[font]['' + ch];
     if (metrics) {
         return {
             depth: metrics[0],
             height: metrics[1],
             italic: metrics[2],
             skew: metrics[3],
-            width: metrics[4],
```

Comment:
`width` was here for experimental reasons.  By default fontMetricsData.js doesn't include the width of the characters, but if you do `make extended_metrics` you can build a copy that does.

Reply:
should this be left in then?

Reply:
I think so? I believe #632 plans to (eventually) add this to master.

Reply:
@edemaine how useful are horizontal metrics after the introduction of @ronkok's SVG code?

Reply:
`\rotatebox` and `\cancelto` cannot be done with CSS trickery, at least not any that I can think of. So it may yet be necessary to know glyph widths and also something equivalent to `makeVList` to find layout widths. 

Other than those two functions, I can think of no uses for horizontal font metrics

Reply:
I guess if we would ever want to support `\width`, \wd`, etc.? (though only approximately)  There are zeros in there now, I don't see why not populate them.  I'm also not sure about advanced `tabular` support (e.g. `\multicolumn) -- it might have use there.